app/components/MovieDetails/Heading/TitleComponent.style.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+import { device } from 'consts/mediaQueries';
+
+export const TitleWrapper = styled.div`
+  padding: 8px 16px;
+  padding-top: 20px;
```

Comment:
you're using padding first and then overwriting it by padding-top.

```suggestion
  padding: 20px 16px 8px;
```